src/transformers/models/vits/configuration_vits.py

```diff
@@ -0,0 +1,213 @@
+# coding=utf-8
+# Copyright 2023 The VITS Authors and the HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" VITS model configuration"""
+
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+VITS_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "Matthijs/mms-tts-eng": "https://huggingface.co/Matthijs/mms-tts-eng/resolve/main/config.json",
```

Comment:
Will do - there are 1000+ checkpoints, so I'll run the conversion script once we've finalised the design and update accordingly (otherwise quite expensive to re-run 😅)

Reply:
Resolved in https://github.com/huggingface/transformers/pull/24085/commits/d07153761337bae0ccffbf959ee61ab27daeaa37